powertools-parameters/src/main/java/software/amazon/lambda/powertools/parameters/DynamoDbProvider.java

```diff
@@ -6,13 +6,8 @@
 import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
-import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
-import software.amazon.awssdk.services.dynamodb.model.GetItemRequest;
-import software.amazon.awssdk.services.dynamodb.model.GetItemResponse;
-import software.amazon.awssdk.services.dynamodb.model.QueryRequest;
-import software.amazon.awssdk.services.dynamodb.model.QueryResponse;
-import software.amazon.lambda.powertools.core.internal.LambdaConstants;
-import software.amazon.lambda.powertools.core.internal.UserAgentConfigurer;
+import software.amazon.awssdk.services.dynamodb.model.*;
```

Comment:
no "*", please expand

Reply:
ohh! I did again! I had configured my IDE to not do that, but I might have missed something. Will fix that.